include/zephyr/drivers/counter.h

```diff
@@ -113,6 +113,35 @@ extern "C" {
 
 /**@} */
 
+/**
+ * @anchor COUNTER_CAPTURE_FLAGS
+ * @name Counter capture flags
+ *
+ * @brief Used by @ref counter_capture_callback_set.
+ * @{
+ */
+
+/**
+ * @brief Capture rising edge of an external input signal
+ */
+ #define COUNTER_CAPTURE_RISING_EDGE BIT(0)
+
+/**
+ * @brief Capture falling edge of an external input signal
+ */
+ #define COUNTER_CAPTURE_FALLING_EDGE BIT(1)
+
+/**
+ * @brief Capture both falling and rising edge of an external input signal
+ */
+ #define COUNTER_CAPTURE_BOTH_EDGES (COUNTER_CAPTURE_FALLING_EDGE | COUNTER_CAPTURE_RISING_EDGE)
+
+ /**@} */
+
+typedef void (*counter_capture_cb_t)(const struct device *dev, uint8_t chan_id,
```

Comment:
Suggestion
```
/** @brief Counter capture callback
 *
 * @param dev       Pointer to the device structure for the driver instance
 * @param chan_id   Channel ID
 * @param flags     Configuration flags (COUNTER_CAPTURE_*)
 * @param ticks     Counter value that triggered the capture
 * @param user_data User data
 */

Reply:
done

Reply:
CI compliance check taints on the space char missing before `*` character above.